05_syntax_lab/02.py

```diff
@@ -1,6 +1,23 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+"""
+Syntax excersize #2
+"""
+sum = 0
+number = 0
+Boom = ""
+rand = 0
+
+from random import randint
+
+while number < 7:
+   rand = randint(1, 100)
+   print rand
+   sum  += rand
+   print sum
+   number += 1
+
+if sum % 7 == 0:
+    Boom = "Boom"
+
+print "The sum of 7 random numbers between 1 and 100 is: {} {}".format(sum, Boom)
+

```

Comment:
I can see why you used the Boom variable, but I think it turned out not that great. I think we can have a more readable version by using two variables for the two text strings and using two separate print statements, something like:

```
if sum % 7 == 0:
  print BOOM.format(...)
else:
  print NOBOOM.format(...)
```

Try to define the variables BOOM and NOBOOM without the duplication and see if you like it better